app/presenters/payment_presenter.rb

```diff
@@ -19,9 +23,28 @@ def refunded_message
     end
   end
 
+  def no_action_message
+    if already_reverted?
+      I18n.t(".reversal_forms.index.already_reversed")
+    else
+      I18n.t(".reversal_forms.index.not_aplicable")
+    end
+  end
+
+  def revertible?
```

Comment:
Another question about language - we have the `reversible` scope but here we use `revertible`. I assume we are looking for the same conditions, but the different terms could be confusing.